demo/batch.R

```diff
@@ -0,0 +1,86 @@
+#TODO: functionalize this? and break out some sections?
+#TODO: tests!
+
+library(dplyr)
+library(loadflex)
+library(rloadest)
+
+#run loadflex over multiple sites
+
+outputFormat <- "simple" #or "complex"
+
+#TODO: read-in function
+load('data/ana_test.Rdata')
+qDF <- ana_discharge
+nutriDF <- as.data.frame(ana_no3, stringsAsFactors = FALSE)
+siteDF <- test_sites
+#one function/format for now
+#needs to convert dates
+nutriDF$date <- as.Date(nutriDF$date)
+qDF$date <- as.Date(qDF$date)
+
+siteSummaries <- data.frame()
+modelSummaries <- data.frame()
+allModels <- list()
+annuals <- data.frame()
+
+#loop over unique sites
+for(site in unique(nutriDF$CODIGO_ESTACAO)){
+  
+  #pull out this site
+  siteNutri <- filter(nutriDF, CODIGO_ESTACAO == site)
+  siteQ <- filter(qDF, CODIGO_ESTACAO == site)
+  #create metadata
+  #not sure units etc are following the correct format
+  siteMeta <- metadata(constituent = "NO3_mg_L", flow = "Q_m3s", dates = "date",
+                       conc.units = "mg/L", flow.units = "m^3/s", load.units = "kg",
+                       load.rate.units = "kg/d", station = site)
+    
+  #TODO: site metrics
+  siteMetrics <- summarizeSites(site, siteDF, siteNutri)
+  
+  #fit models
+  #can we expand getInfo to access the column names in the metadata object?
```

Comment:
I guess I meant here the info in the metadata object that corresponds to column names (i.e. dates, flow, etc).  Since it already needs to specified earlier for the loadflex metadata, it would be nice be able to reference the metadata object here.